packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/User.kt

```diff
@@ -68,6 +69,34 @@ public interface User {
      */
     public val identities: List<UserIdentity>
 
+    /**
```

Comment:
Should all of these fields be treated as public? Not sure why but feels concerning to have the tokens accessible in this manner

Reply:
Yes, they are part of the public API. The tokens are something developers should treat carefully, but they are needed if you e.g. want to call REST API's on App Services yourself.

Also, since the app is running in a secure sandbox, it shouldn't be possible to access them there.